plugin.image.dumpert/addon.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+#
+# Imports
+#
+from future import standard_library
+standard_library.install_aliases()
+from builtins import str
+import os
+import sys
+import urllib.parse
+import xbmc
+import xbmcaddon
+
+LIB_DIR = xbmc.translatePath(
+    os.path.join(xbmcaddon.Addon(id="plugin.image.dumpert").getAddonInfo('path'), 'resources', 'lib'))
+sys.path.append(LIB_DIR)
+
+from dumpert_const import ADDON, DATE, VERSION, SETTINGS
+
+# Parse parameters...
+if len(sys.argv[2]) == 0:
+    #
+    # Main menu
+    #
+    xbmc.log("[ADDON] %s, Python Version %s" % (ADDON, str(sys.version)), xbmc.LOGDEBUG)
+    xbmc.log("[ADDON] %s v%s (%s) is starting, ARGV = %s" % (ADDON, VERSION, DATE, repr(sys.argv)),
+                 xbmc.LOGDEBUG)
+
+    if SETTINGS.getSetting('onlyshownewimagescategory') == 'true':
+        import dumpert_json as plugin
+    else:
+        import dumpert_main as plugin
+else:
+    action = urllib.parse.parse_qs(urllib.parse.urlparse(sys.argv[2]).query)['action'][0]
```

Comment:
this is not python2 compatible... only python3

Reply:
I suppose it'll work due to the future import?

Reply:
it works in both.